gallery/src/PageHeader/PageHeader.scss

```diff
@@ -0,0 +1,50 @@
+.nx-gallery-page-header {
```

Comment:
The gallery's standard classname prefix is just `gallery`, not `nx-gallery`

Reply:
> The gallery's standard classname prefix is just `gallery`, not `nx-gallery`

Thanks, updated: 188ca23b4a36433fcc202fd279284a0dafbf5dbf